src/examples/go_singlerepo.yml

```
@@ -0,0 +1,50 @@
+description: |
+  This example shows:
+   - Specific version of the isopod.
+   - this is for go application
+usage:
+  version: 2.1
+
+  orbs:
+    go: circleci/go@1.7.1
+    ric-orb: ricardo/ric-orb@2
+
+  #FIXME: isopod version that you want to use
+  cfg-isopod-version: &cfg-isopod-version
+    isopod_version: 0.29.4
+
+  workflows:
+    version: 2
+    #FIXME: replace with app name
+    singlerepo-app:
+      jobs:
+        - ric-orb/go_build_test:
+            name: build-test
```

you might have problems if you have no context at all here
```suggestion
        - ric-orb/go_build_test:
            name: build-test
            context: dev
```